base/precompilation.jl

```diff
@@ -1098,7 +1098,7 @@ function _precompilepkgs(pkgs::Vector{String},
             direct = strict ? "" : "direct "
             err_msg = "The following $n_direct_errs $(direct)dependenc$(pluralde) failed to precompile:\n$(String(take!(err_str)))"
             if internal_call # aka. auto-precompilation
-                if isinteractive() && !get(ENV, "CI", false)
+                if isinteractive() && (Base.get_bool_env("CI", false) == false)
```

Comment:
I think the CI check isn't needed... this is all about whether the repl is live, which is satisfied with the first part. (I think I was confused with whether we should be fancy printing, which checks for TTY and not the CI flag, because some CI has a TTY forwarded to a log file)
```suggestion
                if isinteractive()
```